index.css

```diff
@@ -0,0 +1,1205 @@
+    font-style: normal;
+    font-weight: normal;
+    font-size: 32px;
+    color: #091e42;
+  }
+
+  .daily {
+    width: 327px;
+    padding-left: 30px;
+    height: 147px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .botones4 {
+    width: 49px;
+    height: 40px;
+    position: relative;
+    left: 80px;
+    padding: 10px 19px;
+    margin-left: 20px;
+  }
+
+  form ul {
+    position: relative;
+    list-style: none;
+  }
+
+  .botones4 ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    font-size: 0;
+    align-items: center;
+  }
+
+  .botones4 button {
+    background-color: rgba(255, 255, 255, 0.24);
+    color: #091e42;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .proyect {
+    display: flex;
+    justify-content: start;
+    width: 377px;
+  }
+
+  .proyect button {
+    width: 100px;
+    height: 40px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    background: #ff6b00;
+    color: #fff;
+    margin-left: 43px;
+  }
+
+  .bottom-left {
+    position: absolute;
+    bottom: 8px;
+    left: 16px;
+  }
+
+  .dashbord {
+    position: relative;
+    bottom: 330px;
+    left: 40px;
+    width: 295px;
+    height: 81px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  .texto3 {
+    position: relative;
+    width: 295px;
+    height: 87px;
+    bottom: 340px;
+    left: 40px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+  }
+
+  .botones3 {
+    position: relative;
+    bottom: 340px;
+    margin-right: 90px;
+  }
+
+  .botones3 button {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    background-color: rgba(255, 255, 255, 0.24);
+    color: #fff;
+    margin-left: 5px;
+  }
+
+  .botones3 ul {
+    display: flex;
+    font-size: 0;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .brian button {
+    position: relative;
+    bottom: 323px;
+    left: 29px;
+    width: 327px;
+    height: 60px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    background-color: #ff6b00;
+    font-size: 17px;
+    color: #fff;
+  }
+
+  .container {
+    height: 400px;
+  }
+
+  .jo {
+    position: relative;
+    height: 400px;
+  }
+
+  .ran {
+    position: relative;
+    height: 400px;
+  }
+
+  .ad {
+    position: relative;
+    height: 400px;
+  }
+
+  .ria {
+    position: relative;
+    height: 400px;
+  }
+
+  .ald {
+    position: relative;
+    height: 40px;
+  }
```

Comment:
[Optional]
- [ ] You have 5 different classes doing exactly the same thing,  the whole reason behind the usage of the class structure is not repeating ourselves again and again. Your code will be more readable and maintainable if you use the same class for multiple elements.
(I gave these 5 classes as an example, the code has a lot of similar cases)